### ISSUE TITLE
Select - Enhance Contrast (Enhanced) AAA

### ISSUE THREAD

**Issue by julauxen** [grommet/grommet#7621]
In Select, placeholder text against background has a 4.61:1 contrast ratio. Option text against hover background has a 6.84:1 contrast ratio.
Both fail WCAG rule 1.4.6 Contrast (Enhanced) AAA that requires a 7:1 ratio.

**Comment by jcfilben** [comment]
Closing as not planned, we are unlikely to alter our theme colors for AAA at this point in time
